awscli/customizations/emr/helptext.py

```diff
@@ -205,7 +205,8 @@
     ' are used for the core node type.</p>')
 
 ADDITIONAL_INFO = (
-    '<p>Specifies additional information during cluster creation.</p>')
+    '<p>Specifies additional information during cluster creation. To set development mode when starting your EMR cluster,'
+    ' set this parameter to <code>{"clusterType":"development"}</code>.</p>'
```

Comment:
Oh I think it is missing the ending parenthesis (e.g. `)`) now, which is causing the tests to fail. So line 209 should be just:
```
' set this parameter to <code>{"clusterType":"development"}</code>.</p>')
```

Reply:
My bad - wasn't careful in my last commit.